### ISSUE TITLE
New release of scalamock-scalatest-support

### ISSUE THREAD

**Issue by fedeoasi** [paulbutcher/ScalaMock#105]
I see that the latest release of scalamock-scalatest-support on maven central has a dependency to scalatest 2.1.3:
http://central.maven.org/maven2/org/scalamock/scalamock-scalatest-support_2.11/3.2.1/scalamock-scalatest-support_2.11-3.2.1.pom

The current code depends on scalatest 2.2.4:
https://github.com/paulbutcher/ScalaMock/blob/master/project/Build.scala#L95

Is it possible to get a release out so that we can upgrade scalatest to 2.2.4 in our projects? Thanks!

**Comment by AlexeyRaga** [comment]
+1 for getting a new release after #104 is merged as both these concerns are important to us.

**Comment by pawel-wiejacha** [comment]
I'm going to release new ScalaMock version today with #104 and #106 included. Also, the new release will use scalatest-2.2.4.

@fedeoasi you don't have have to wait for new scalamock-scalatest-support release. You can add `scalatest-2.2.4` to your `libraryDependencies` to use ScalaTest 2.2.4. For example:

`[build.sbt]`:
```scala
scalaVersion := "2.11.1"

libraryDependencies ++= Seq(
    "org.scalamock" %% "scalamock-scalatest-support" % "3.2.1" % "test",
    "org.scalatest" %% "scalatest" % "2.2.4" % "test")
```

`sbt $ show update`:
```
[info]  test:
[info]  org.scalatest:scalatest_2.11
[info]          - 2.2.4
[info]                  [...]       
[info]                  evicted: false
[info]                  [...]       
[info]          - 2.1.3
[info]                  evicted: true
```

**Comment by pawel-wiejacha** [comment]
Fixed in cf8aa67cf163682cd52cbb4339e118da47703ae3
